README.md

```diff
@@ -22,13 +10,81 @@ USAGE:
 clj-watson scan [command options] [arguments...]
 
 OPTIONS:
--p, --deps-edn-path S*                       path of deps.edn to scan
--d, --dependency-check-properties S          path of deps.edn to scan
--o, --output edn|json|stdout         stdout  Output type
+-p, --deps-edn-path S*                       path of deps.edn to scan.
+-d, --dependency-check-properties S          path of a dependency-check properties file.
+-o, --output edn|json|stdout         stdout  Output type.
```

Comment:
✂️ duplicated `stdout`

Reply:
it's because stdout is the default option.